include/networkit/generators/MocnikGenerator.hpp

```diff
@@ -62,18 +62,18 @@ class MocnikGenerator: public StaticGraphGenerator {
 	/**
 	 * Add the node with number j to the corresponing grid cell
 	 */
-	const void addNode(LayerState &s, const int &j);
+	void addNode(LayerState &s, const int &j);
 
 	/**
 	 * Determine, for a given position, the index of the corresponding grid cell
 	 */
-	const int toIndex(LayerState &s, const std::vector<double> &v);
+	int toIndex(LayerState &s, const std::vector<double> &v);
 
 	/**
 	 * Determine, for the given multi-dimensional index, the index of the
 	 * corresponding grid cell
 	 */
-	const int toIndex(LayerState &s, const std::vector<int> &v);
+	int toIndex(LayerState &s, const std::vector<int> &v);
```

Comment:
(out of scope of this PR?) Move const to end of line and make it const this? Alsso s can be a const ref

Reply:
Out of scope.